packages/plugin-bundler-webpack/src/webpack.config.js

```diff
@@ -3,22 +3,27 @@ import path from "path";
 import webpack from "webpack";
 import ExtractTextPlugin from "extract-text-webpack-plugin";
 
+const isStatic = process.env.PHENOMIC_ENV === "static";
+const isProduction = process.env.NODE_ENV === "production";
```

Comment:
You cannot put this statically. Must be in the fonction to be safe (since we can adjust this during a function - we are not 100% sure that this file won't be imported (& so evaluated) before the code that will adjust this on runtime - FYI [here](https://github.com/phenomic/phenomic/blob/f03a7398322f4f8c56ef86353a11de5f1c4f435d/packages/core/src/commands/build.js#L122-L124) and [here](https://github.com/phenomic/phenomic/blob/f03a7398322f4f8c56ef86353a11de5f1c4f435d/packages/core/src/commands/start.js#L41-L43))